client/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-submit-button.tsx

```diff
@@ -0,0 +1,108 @@
+import { StripeConfiguration } from '@automattic/calypso-stripe';
+import { Button, FormStatus, useFormStatus } from '@automattic/composite-checkout';
+import { useElements, CardElement } from '@stripe/react-stripe-js';
+import { Stripe } from '@stripe/stripe-js';
+import { useSelect } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import debugFactory from 'debug';
+import { useMemo } from 'react';
+import { State } from 'calypso/state/partner-portal/payment-methods/reducer';
+
+const debug = debugFactory( 'calypso:partner-portal:credit-card' );
+
+export default function CreditCardSubmitButton( {
+	disabled,
+	onClick,
+	store,
+	stripe,
+	stripeConfiguration,
+	activeButtonText,
+}: {
+	disabled?: boolean;
+	onClick?: ( element: string, submitData: unknown ) => void;
+	store: State;
+	stripe: Stripe | null;
+	stripeConfiguration: StripeConfiguration | null;
+	activeButtonText: string | undefined;
+} ): JSX.Element {
+	const { __ } = useI18n();
+	const fields = useSelect( ( select ) => select( 'credit-card' ).getFields() );
+	const useAsPrimaryPaymentMethod = useSelect( ( select ) =>
+		select( 'credit-card' ).useAsPrimaryPaymentMethod()
+	);
+	const cardholderName = fields.cardholderName;
+	const { formStatus } = useFormStatus();
+	const elements = useElements();
+	const cardElement = elements?.getElement( CardElement ) ?? undefined;
+	const formSubmitting = FormStatus.SUBMITTING === formStatus;
+
+	const buttonContents = useMemo( () => {
+		if ( formStatus === FormStatus.SUBMITTING ) {
+			return __( 'Processing…' );
+		}
+		if ( formStatus === FormStatus.READY ) {
+			return activeButtonText || __( 'Save payment method' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Saved payment method' )` *ES Score: 8*